Runtime/Core/ShapeGenerator.cs

```diff
@@ -1562,7 +1562,7 @@ public static ProBuilderMesh GenerateTorus(PivotLocation pivotType, int rows, in
 
             int col = clampedColumns - 1;
 
-            Vector3[] cir = GetCirclePoints(clampedRows, clampedTubeRadius, clampedVerticalCircumference, Quaternion.Euler(Vector3.up * 0f * clampedHorizontalCircumference), clampedRadius);
+            Vector3[] cir = GetCirclePoints(clampedRows, clampedTubeRadius, clampedVerticalCircumference, Quaternion.Euler(0f * clampedHorizontalCircumference * Vector3.up), clampedRadius);
```

Comment:
I understand that this is just changing the order of operations to multiply single values before vec3, but why are we multiplying `0f * clampedHorizontalCircumference * Vector3.up` in the first place? Isn't this equivalent to simply writing `vec3(0,0,0)`?

Reply:
Good question and no idea how I overlooked this (or why it was there in the first place). Fixed in new commit